client/my-sites/beta-testing/main.jsx

```diff
@@ -0,0 +1,188 @@
+import { useTranslate } from 'i18n-calypso';
+import { flowRight } from 'lodash';
+import FollowButtonContainer from 'calypso/blocks/follow-button';
+import Gridicon from 'calypso/components/gridicon';
+
+/**
+ * Internal dependencies
+ */
+import Main from 'calypso/components/main';
+import { Button } from '@automattic/components';
+import SidebarNavigation from 'calypso/my-sites/sidebar-navigation';
+import { getSelectedSite, getSelectedSiteId } from 'calypso/state/ui/selectors';
+import PageViewTracker from 'calypso/lib/analytics/page-view-tracker';
+import DocumentHead from 'calypso/components/data/document-head';
+import QuerySiteChecklist from 'calypso/components/data/query-site-checklist';
+import { bumpStat, composeAnalytics, recordTracksEvent } from 'calypso/state/analytics/actions';
+import {
+	getPostsForQueryIgnoringPage,
+	isRequestingPostsForQueryIgnoringPage,
+} from 'calypso/state/posts/selectors';
+import QueryPosts from 'calypso/components/data/query-posts';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+import CardHeading from 'calypso/components/card-heading';
+
+const query = {
+	author: null,
+	number: 3,
+	order: 'DESC',
+	status: 'publish',
+	type: 'post',
+	category: 'testing',
+};
+const horizonSiteId = 90972941;
+
+const BetaTesting = ( { siteId, posts, isRequestingPosts, trackViewHorizonAction } ) => {
+	const translate = useTranslate();
+
+	const main = (
+		<section className="beta-testing">
+			<div className="beta-testing__main-body">
+				<div className="beta-testing__main-info">
+					<CardHeading className="beta-testing__main-title" size={ 36 }>
+						{ translate( 'Help shape the future of online publishing' ) }
+					</CardHeading>
+
+					<p>
+						{ translate(
+							'WordPress.com ships updates and improvements every single day, and beta users will get to experience some of these before anyone else. By taking part in this program, you will be helping to shape the future of WordPress.com, making it better for millions of people worldwide.'
+						) }
+					</p>
+					<p>
+						{ translate(
+							"You won't be testing everything but there are some important ideas and projects we’d love to share with you."
+						) }
+					</p>
+
+					<div className="beta-testing__cta-section">
+						<Button
+							className="beta-testing__view-horizon-button is-primary"
+							href="https://horizon.wordpress.com/"
+							onClick={ trackViewHorizonAction }
+						>
+							{ translate( 'Join the Beta Program' ) }
+						</Button>
+
+						<a
+							className="beta-testing__external-link"
+							href="https://horizonfeedback.wordpress.com/about/"
+						>
+							{ translate( 'Learn how it works' ) }
+							<Gridicon icon="external" size={ 18 } />
+						</a>
+					</div>
+				</div>
+				<div className="beta-testing__main-image-wrapper">
+					<img
+						className="beta-testing__main-image"
+						src="/calypso/images/illustrations/illustration-404.svg"
+						alt={ translate( 'Beta Testing' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **122** times:
`translate( 'Beta Testing', { context: 'Plugin Installer'} )` *ES Score: 16*